src/main/java/uk/gov/hmcts/reform/managecase/service/noc/NoticeOfChangeQuestions.java

```diff
@@ -30,34 +30,40 @@
 import java.util.Optional;
 import java.util.stream.Collectors;
 
+import static java.util.stream.Collectors.toList;
 import static uk.gov.hmcts.reform.managecase.api.errorhandling.ValidationError.CASE_ID_INVALID;
 import static uk.gov.hmcts.reform.managecase.api.errorhandling.ValidationError.CASE_NOT_FOUND;
 import static uk.gov.hmcts.reform.managecase.api.errorhandling.ValidationError.CHANGE_REQUEST;
 import static uk.gov.hmcts.reform.managecase.api.errorhandling.ValidationError.INSUFFICIENT_PRIVILEGE;
 import static uk.gov.hmcts.reform.managecase.api.errorhandling.ValidationError.NOC_EVENT_NOT_AVAILABLE;
 import static uk.gov.hmcts.reform.managecase.api.errorhandling.ValidationError.NOC_REQUEST_ONGOING;
 import static uk.gov.hmcts.reform.managecase.api.errorhandling.ValidationError.NO_ORG_POLICY_WITH_ROLE;
+import static uk.gov.hmcts.reform.managecase.repository.DefaultDataStoreRepository.CHANGE_ORGANISATION_REQUEST;
 
 @Service
-@SuppressWarnings({"PMD.DataflowAnomalyAnalysis", "PMD.ExcessiveImports",
+@SuppressWarnings({"PMD.DataflowAnomalyAnalysis",
+    "PMD.GodClass",
+    "PMD.ExcessiveImports",
+    "PMD.TooManyMethods",
     "PMD.AvoidDeeplyNestedIfStmts", "PMD.PreserveStackTrace", "PMD.LawOfDemeter",
     "PMD.AvoidLiteralsInIfCondition", "PMD.CyclomaticComplexity"})
```

Comment:
Why would we need so many suppressions? Are they actually all in effect, or just residual from momentary additions?

Reply:
Fixed in https://github.com/hmcts/aac-manage-case-assignment/pull/150/commits/04ca3c6315a335ded9f9683f351c1dbf78f7d120